code/src/Providers/Google/DrawingTools/AbstractDrawShape.ts

```diff
@@ -33,7 +33,8 @@ namespace Provider.Google.DrawingTools {
             // Add the onChange event to the new element
             this.setOnChangeEvent(_shape, _shape.config.locations);
             // Add the new element to the map
-            this.map.addShape(_shape);
+            OutSystems.Maps.MapAPI.ShapeManager.CreateDrawedShape(_shape);
```

Comment:
Hum I don't think so, because we don't have an outsystems block for these shapes. These are created in runtime using the drawingTool. And we want them to be added to our ShapeManager API, that is kind of a different thing in our framework.

Do you have any ideia?